app/webpacker/components/Schedule/VenuesAndRooms.jsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import {
+  Form, Grid, Menu, Message,
+} from 'semantic-ui-react';
+import { getTextColor } from '../../lib/utils/calendar';
+import { toDegrees } from '../../lib/utils/edit-schedule';
+import i18n from '../../lib/i18n';
+
+export default function VenuesAndRooms({
+  venues,
+  activeVenueOrNull,
+  activeVenueIndex,
+  setActiveVenueIndex,
+  timeZoneCount,
+  rooms,
+  activeRoomIds,
+  dispatchRooms,
+}) {
+  const venueCount = venues.length;
+
+  const setActiveVenueIndexAndResetRooms = (newVenueIndex) => {
+    const newVenues = newVenueIndex > -1 ? [venues[newVenueIndex]] : venues;
+    const ids = newVenues.flatMap((venue) => venue.rooms).map((room) => room.id);
+    dispatchRooms({ type: 'reset', ids });
+
+    setActiveVenueIndex(newVenueIndex);
+  };
+
+  return (
+    <>
+      {venueCount > 1 && (
+        <Menu
```

Comment:
This looks weird now, there's no underline of the selected venue, so it's less clear that this is a menu.

![image](https://github.com/thewca/worldcubeassociation.org/assets/49137025/c3ac1f96-6eb5-40c4-ae32-b4c26ff09854)

Compare to calendar vs table menu:

![image](https://github.com/thewca/worldcubeassociation.org/assets/49137025/45530302-de79-4db0-9c1c-8f27b5897049)